terraform/cb.tf

```diff
@@ -0,0 +1,55 @@
+data "google_project" "project" {
+}
```

Comment:
should this explicityl reference `var.secrets_project_id` because it is used for github PAT secret?